core/jobs/batch_jobs/suggestion_edit_state_content_deletion_jobs_test.py

```diff
@@ -0,0 +1,250 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.batch_jobs.suggestion_edit_state_content_deletion_jobs.
+"""
+
+from __future__ import annotations
+
+from core import feconf
+from core.jobs import job_test_utils
+from core.jobs.batch_jobs import suggestion_edit_state_content_deletion_jobs
+from core.jobs.types import job_run_result
+from core.platform import models
+
+from typing import Type
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import suggestion_models
+
+(suggestion_models, ) = models.Registry.import_models([
+    models.Names.SUGGESTION
+])
+
+
+class DeleteDeprecatedSuggestionEditStateContentModelsJobTests(
+    job_test_utils.JobTestBase):
+
+    dj = suggestion_edit_state_content_deletion_jobs
```

Comment:
Avoid using short names or maybe make them a bit more descriptive.

Reply:
done, made it more descriptive